docs/docs/guides/05-type-system/02-concepts/01-schema.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Schema
+tags:
+  - typescript
+  - type system
+  - type
+  - concepts
+---
+
+The schema is the primary data structure leveraged within the Strapi Type System, defining how content is structured and managed in the application.
+
+It serves several key functions:
+
+- **Representation**: At its core, a schema outlines and defines the structure of Strapi content. This is useful when dealing with features that need access to low level schema properties (_e.g. attributes, plugin options, etc..._).
+
+- **Inference**: The schema allows inferring and configuring numerous other types. This includes entities like `ContentType` or `Component`, among others.
+
+### Scope
+
+Schema types represent **loaded** schemas in the context of a Strapi server application and should be used accordingly.
+
+:::caution
+Database models and schema definitions (before being loaded by Strapi) are not the same types and can't be used interchangeably.
+:::
+
+### Sub-Types
+
+Each box is a type that extends the base Schema interface.
+
+In between each box is text that represents the discriminant used to differentiate the subtype from others.
+
+```mermaid
+flowchart TB;
+  Schema -- "<code>modelType: contentType</code>" ---- ContentTypeSchema
+  Schema -- "<code>modelType: component</code>" ----  ComponentSchema
+  ContentTypeSchema -- "<code>kind: collectionType</code>" ---- CollectionTypeSchema
+  ContentTypeSchema -- "<code>kind: singleType</code>" ---- SingleTypeSchema
+```
+
+### Properties
+
+Schema types contain useful information that helps other types know how to interact with the Strapi content.
+
+This is facilitated through multiple properties.
+
+#### Options
+
+A set of properties used to configure the schema. It contains information on features activation among other things.
+
+This can be really useful to make the types adapt to a given schema.
+
+For instance, the document service uses the `options.draftAndPublish` property to determine whether it should add publication methods to the service type.
+
+#### Plugin Options
+
+These options provide the ability to alter or enhance the behaviour of the system based on specific values.
+
+If a plugin is enabled, it might bring functionality that can affect how types interact with each other.
+
+For example, it's possible to add or remove certain entity-service filters from the query type based on whether a plugin is enabled.
+
+#### Attributes
+
+Strongly typed schema attribute allows the Type System to infer actual entities types based on their properties.
```

Comment:
```suggestion
Strongly type schema attributes allow the Type System to infer actual entities types based on their properties.
```

Reply:
In this context, we are referring to the schema attributes being strongly typed, but it's missing an 's' on attribute indeed

Reply:
```suggestion
Strongly typed schema attributes allows the Type System to infer actual entities types based on their properties.
```